app/components/common/Countdown.component.js

```diff
@@ -1,7 +1,7 @@
 import React, { useEffect, useState } from "react";
 
-const challengeStartDate = new Date(2021, 4, 31, 0, 0, 0, 0); // 31st May
-const challengeEndDate = new Date(2021, 6, 1, 0, 0, 0, 0); // 1st July 
+const challengeStartDate = new Date(2021, 5, 21, 0, 0, 0, 0); // 21st June
```

Comment:
Did we say sign up on the 21st and then challenge start on the 22nd? Or would sign ups be blocked unless we have the start date on the 22nd June?

Reply:
I have confirmed this date with Nick today.